automated/linux/imagenet-resnet50/imagenet-resnet50.yaml

```diff
@@ -0,0 +1,24 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: imagenet-resnet50
+    description: "imagenet-resnet50 job, running resnet50 benchmark through mlperf."
+    maintainer:
+        - theodore.grey@linaro.org
+    os:
+        - debian
+    scope:
+        - functional
+    devices:
+        - hip07-d05
+params:
+    HOME_DIR: '/home/debiand05'
+    MNT_DIR: '/mnt'
+    MNT_EXISTS: true
+    SKIP_INSTALL: false
+    TEST_PROG_VERSION: ""
+    TEST_GIT_URL: ""
+    TEST_DIR: ""
```

Comment:
you need to set the variable here too, if no the 'get_test_program()' will return that it assumes its already installed.